### ISSUE TITLE
Issuing monitor string PVs from pcaspy

### ISSUE THREAD

**Issue by ddamiani** [paulscherrerinstitute/pcaspy#24]
I noticed a strange issue when monitoring string PVs created by pcaspy. The monitors get a callback as expected when the string changes however the value returned is not expected, It always seems to be a string representation of decimal number like '0.000' or '7.000'.

Doing a caget always returns the expected value.

**Comment by xiaoqiangwang** [comment]
I could verify this problem with the following example

```
prefix = 'MTEST:'
pvdb = {
    'RAND' : {
        'type': 'str'
    },
}

class myDriver(Driver):
    def  __init__(self):
        super(myDriver, self).__init__()

```

```
$ caput MTEST:RAND ui
Old : MTEST:RAND                     
New : MTEST:RAND                     ui

$ caput MTEST:RAND 9
Old : MTEST:RAND                     ui
New : MTEST:RAND                     9
```

The output from _camonitor_
```
$ camonitor -t n MTEST:RAND
MTEST:RAND                       UDF INVALID
MTEST:RAND                      0.0000 UDF INVALID
MTEST:RAND                      9.0000 UDF INVALID
```

**Comment by xiaoqiangwang** [comment]
This is fixed now in commit 091d474. It is a bug introduced from 0.6.1.

**Comment by ddamiani** [comment]
Thank you the fix worked for me.
